### DISCUSSION TITLE
Need Help Setting up Security Onion in VM

### DISCUSSION THREAD

**Discussion by csull247** [Security-Onion-Solutions/securityonion#9560] [General] [answered]
I am attempting to set up Security Onion in my VM and continue to receive an error message (see attached screen shot). Can someone guide me in the right direction. I am new to cybersecurity and this is an on going learning process for me. If additional information is needed, please request and I will do my best to provide it. Also, I am not sure if this question has been asked, I did conduct a general search and I was not able to find a match. Thank you. 

![Screenshot_20230111_102448](https://user-images.githubusercontent.com/122471712/211871628-41044a31-b60e-47e8-8bc8-b23d42f48d62.png)

**Comment by TotieBash** [comment]
It looks like it is not detecting the SO CentOS iso image.  You need to attach the Centos iso image.
https://github.com/Security-Onion-Solutions/securityonion/blob/master/VERIFY_ISO.md

**Reply by csull247** [reply]
Thanks for the reply. I accessed the link you provided, and ultimately downloaded the image again. From the screen shot I provided, I clicked the drop down "dvd" and selected the ISO, and it seemed to work from there. Also, I noticed when I changed the boot order, which was first checked "floppy" to "disc image" it worked. My question, from the link you sent, when they say to "download and verify" how do you actually verify the download? Lastly, I have completed most of the set-up, but I have reached a part that states "Warning, installing the security onion analyst node will make permanent changes to the system." Should I proceed from here?

**Comment by InfosecGoon** [comment, marked as answer]
Here's a good walkthrough on booting from the ISO in Virtualbox -- you'll need to do that in order to build the VM.

https://onlinecomputertips.com/support-categories/software/662-virtualbox-boot-iso/

**Reply by csull247** [reply]
Thanks for the reply!

**Reply by csull247** [reply]
Question for you: I am entering my third semester of cybersecurity. At times, I feel like I am not understanding a lot of the basic concepts that are being taught. I have never been a "tech" guy, so this may be adding to the confusion. My initial thought with taking on cybersecurity was the awesome awaiting job prospect, and it sorta aligns with what I do now. I am wanting to change careers and need to make an income similar to what I have now. But I feel like their is a ton to learn and I am not catching on fast enough. I often wonder if I should experience another field of technology to lay a foundation for learning cybersecurity easier? I have also heard that cybersecurity is more "back-end" stuff, perhaps this means more complicated and more to learn? I have always been a hands-on type of person, so I think this adds to the confusion in learning technology since I cannot actually see how a process works, if that makes sense. I would appreciate your feedback! Thanks.

**Reply by InfosecGoon** [reply]
With the caveat that I am by no means a career counselor, in my (anecdotal) experience people have more success moving to a security role from another information technology role than they do just trying to jump into security right off the bat. I think the reason for that is that so much of security is about making the proper choices and tradeoffs in configurations and operational decisions -- if you don't understand how things like operating systems or network traffic work on a deep level, you're not going to have the necessary context to evaluate whether they're secure or not.

**Comment by LayerIIX** [comment]
I had this problem and if I remember correctly the "Secure boot" setting was enabled and so it kept kicking out the Security Onion ISO.  Make sure the secure boot setting is unchecked in the VM configuration and try again.

**Reply by csull247** [reply]
I will try this.. thanks.
